DNN Platform/Components/ClientDependency/Source/Controls/ClientDependencyInclude.cs

```diff
@@ -58,31 +60,27 @@ protected ClientDependencyInclude(IClientDependencyFile file)
         /// </summary>
         public bool ForceBundle { get; set; }
 
-        /// <summary>Used to store additional attributes in the HTML markup for the item</summary>
-        /// <remarks>Mostly used for CSS Media, but could be for anything</remarks>
+        /// <summary>
+        /// Used to store additional attributes in the HTML markup for the item
+        /// </summary>
+        /// <remarks>
+        /// Mostly used for CSS Media, but could be for anything
+        /// </remarks>
         public IDictionary<string, string> HtmlAttributes { get; private set; }
 
-        /// <summary>Used to set the HtmlAttributes on this class via a string which is parsed</summary>
-        /// <remarks>The syntax for the string must be: <c>key1:value1,key2:value2</c> etc.</remarks>
+        /// <summary>
+        /// Used to set the HtmlAttributes on this class via a string which is parsed
+        /// </summary>
+        /// <remarks>
+        /// The syntax for the string must be: key1:value1,key2:value2   etc...
+        /// </remarks>
         public string HtmlAttributesAsString { get; set; }
 
         protected override void OnPreRender(EventArgs e)
         {
             base.OnPreRender(e);
             if (string.IsNullOrEmpty(FilePath))
-            {
                 throw new NullReferenceException("Both File and Type properties must be set");
-            }
-        }
-
-        protected override void Render(HtmlTextWriter writer)
-        {
-            if (AddTag || this.Context.IsDebuggingEnabled)
-            {
-                writer.Write("<!--CDF({0}|{1})-->", DependencyType, FilePath);

```

Comment:
We'll want to add this piece back, it's an addition that DNN uses when a module is cached

Reply:
Code restored.